contrib/cni/README.md

```diff
@@ -28,7 +28,7 @@ Copy the respective files to the configuration directory like so:
 sudo cp 10-crio-bridge.conflist /etc/cni/net.d
 ```
 
-By default, we install the dual stack version: [10-crio-bridge.conflist][dual-stack]
+By default, we install the IPv4 only version: [11-crio-ipv4-bridge.conflist][ipv4-only]
```

Comment:
@cri-o/cri-o-maintainers would it be better to install the dual stack config per default in the packages?

Reply:
yeah I think installing the dual stack one in the packaging would be better

Reply:
> yeah I think installing the dual stack one in the packaging would be better

Then let's close this PR.